Installation/test/Installation/CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ function(CGAL_installation_test_find_package_version mode)
 
   configure_file(test_find_package.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/test_find_package_version_${mode}/CMakeLists.txt)
   add_test(NAME test_find_package_version_${mode}
-    COMMAND ${CMAKE_COMMAND} -DCMAKE_BUILD_TYPE=Release ${CMAKE_CURRENT_BINARY_DIR}/test_find_package_version_${mode}
+    COMMAND ${CMAKE_COMMAND} $ENV{CMAKE_GENERATOR} -C$ENV{INIT_FILE} -DCMAKE_BUILD_TYPE=Release ${CMAKE_CURRENT_BINARY_DIR}/test_find_package_version_${mode}
```

Comment:
`-C$ENV{INIT_FILE}` will create an error if `INIT_FILE` is not in the environment.

Reply:
I have seen that. Maybe a genex could help me, I'll dig further...